site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -134,7 +134,7 @@ You may find yourself repeating yourself when you build your kickstart file, nob
 
 To use a variable, define the key and value in the `variables` section of the kickstart file and then access that value anywhere outside of the `variables` section using the `#{foo}` notation where `foo` is the name of the variable.
 
-Here is the same example as above, but instead of hard coding the API key, we are using a variable which can be reused. This example is not particularly helpful, we've only moved the value from the `apiKeys` section to the `variables` section, but using variables can come in handy as you'll see in the next few examples.
+Here is the same example as above, but instead of hard coding the API key, the API key value is a variable which can be reused. This example is not particularly interesting since the value has been moved from the `apiKeys` section to the `variables` section. Never worry! Using variables can come in handy as you'll see in the next few examples.
```

Comment:
```suggestion
Here is the same example as above, but instead of hard coding the API key, a variable is created to hold the API key value. This example is not particularly interesting since the value has been moved from the `apiKeys` section to the `variables` section. Never worry! Using variables can come in handy as you'll see in the next few examples.
```